res/values-in/strings.xml

```diff
@@ -11,17 +11,17 @@
     <string name="WEEKLY">"MINGGUAN"</string>
     <string name="DAILY">"HARIAN"</string>
     <string name="Server_">"Server:"</string>
-    <string name="Game_Mode_">"Mode Permainan:"</string>
+    <string name="Game_Mode_">"Mode Permainan"</string>
     <string name="Show_Names">"Tampilkan Nama"</string>
     <string name="Show_Avatars">"Tampilkan Skin"</string>
     <string name="PLAY">"MAIN"</string>
     <string name="OPTIONS">"OPSI"</string>
 
-    <string name="Avatar_">"Skins:"</string>
-    <string name="Name_">"Nama:"</string>
+    <string name="Avatar_">"Skins"</string>
+    <string name="Name_">"Nama"</string>
```

Comment:
Please return the colons here.

Reply:
this needs to be translated
![Screenshot_2024-12-30-11-53-14-718_software simplicial nebulous](https://github.com/user-attachments/assets/c165133d-2e46-4117-ac6d-c21c41e8b3a0)

Reply:
item PLASMA_BOOST_3X_1D_XMAS/0 not found.

item PLASMA_BOOST_3X_1D_NATAL/0 tidak ditemukan.

Reply:
@Khoirun0524

Item names are not supposed to be translated. If you have some untranslated text to report, please use the [issue tracker](https://github.com/simplicialsoftware/nebulous-translations/issues).